docs/changelog.md

```diff
@@ -31,6 +31,49 @@ permalink: /changelog/
 * **feature-prompts**, **browser-engine-gecko***
   * Added support for [Pop-up windows dialog](https://support.mozilla.org/en-US/kb/pop-blocker-settings-exceptions-troubleshooting#w_what-are-pop-ups).
 
+* **service-firefox-accounts**
+  * Added `FxaAccountManager`, which encapsulates a lower level accounts API and provides an observable interface for consumers that wish to be notified of account and profile changes.
+  * Background-worker friendly.
+  ```kotlin
+  // Long-lived instance, pinned on an application.
+  val accountManager = FxaAccountManager(context, Config.release(CLIENT_ID, REDIRECT_URL), arrayOf("profile"))
```

Comment:
@eoger @grigoryk can anything be done in the FxaAccountManager right now to lay the foundation for the upcoming push / send-tab work? For example, can we briefly outline the events that SendTab <--> Push pipes will handle and brainstorm that?

Reply:
Yes; let’s iterate on this in the tree once the first pass lands.

Reply:
Reason for landing first then iterating with push-specific events/states is that I need to start landing sync-related pieces that make use of this work (and are mostly ready to go), and they're currently higher priority than send tab stuff. Happy to talk about the push/sendtab next week.